src/ontology/cl-edit.owl

```diff
@@ -22034,6 +22039,13 @@ AnnotationAssertion(rdfs:label obo:CL_0010016 "collar cell")
 SubClassOf(obo:CL_0010016 obo:CL_0000548)
 SubClassOf(obo:CL_0010016 ObjectSomeValuesFrom(obo:BFO_0000051 obo:GO_0005929))
 
+# Class: obo:CL_0010017 (zygote)
+
+AnnotationAssertion(obo:IAO_0000115 obo:CL_0010017 "A zygote in a plant or an animal.")
+AnnotationAssertion(rdfs:label obo:CL_0010017 "zygote"@en)
+AnnotationAssertion(rdfs:seeAlso obo:CL_0010017 <https://github.com/obophenotype/cell-ontology/issues/786>)
```

Comment:
In GO we're using:

 [term tracker item](http://purl.obolibrary.org/obo/IAO_0000233)  `"https://github.com/obophenotype/cell-ontology/issues/786"^^xsd:anyURI`

standardize?

Reply:
ok - let's do as separate PR